bodhi/server/models.py

```diff
@@ -1599,7 +1603,9 @@ class Update(Base):
     __get_by__ = ('alias',)
 
     autokarma = Column(Boolean, default=True, nullable=False)
+    autotime = Column(Boolean, default=False, nullable=False)
```

Comment:
It is inconsistent to make this one default to ```False``` when the ```autokarma``` field defaults to ```True```.

```suggestion
    autotime = Column(Boolean, default=True, nullable=False)
```